conn/gpio/gpiosmoketest/gpiosmoketest.go

```diff
@@ -478,19 +479,3 @@ func since(start time.Time) string {
 	µs %= 1000
 	return fmt.Sprintf("%3d.%03dms", ms, µs)
 }
-
-// loggingPin logs when its state changes.
-type loggingPin struct {
-	gpio.PinIO
-	start time.Time
-}
-
-func (p *loggingPin) In(pull gpio.Pull, edge gpio.Edge) error {
-	fmt.Printf("    %s %s.In(%s, %s)\n", since(p.start), p, pull, edge)
```

Comment:
In particular, this is using fmt.Printf, not log, and this is intentional since it is not general logging but really structured output. I just realized I removed them from the CI system by accident, will add this back.